ruby_on_rails/rails_basics/deployment.md

```diff
@@ -46,34 +46,34 @@ The first and most crucial thing PaaS providers give you are virtual "computers"
 
 The cool thing about Rails is that you can always fire up more instances of your application if you get too much traffic and users have to wait for their requests to be filled.
 
-For most of your apps, one instance is plenty enough. You can support a lot of traffic using just a single instance. Many of the PaaS providers we will recommend later in this lesson give you your first one for free.
+For most of your apps, one instance is plenty. You can support a lot of traffic using just a single instance. Many of the PaaS providers we will recommend later in this lesson give you your first one for free.
 
 #### Databases
 The second most important thing PaaS providers give you is databases. They make it easy to spin up a new database for each app by doing all the setup and configuration for you.
 
 Many providers even manage the database for you by setting up automatic backups, ensuring the database is constantly updated with the latest critical security patches and ongoing maintenance that keeps your databases up and running smoothly.
 
-The peace of mind this affords you can't be overstated. You never want to be in a position where you're waking at 4 am to multiple alerts. Your database is on the fritz because of some security patch you forgot to apply, and there are no backups to fall back on.
+The peace of mind this affords you can't be overstated. You never want to be in a position where you're waking at 4 a.m. to multiple alerts. Your database is on the fritz because of some security patch you forgot to apply, and there are no backups to fall back on.
 
-With Rails, we will be using PostgreSQL, a popular open-source database. Some PaaS providers like Heroku will automatically create a PostgreSQL database for our application when we first deploy. Others, like Fly.io, will have a few more manual steps involved, but still beats having to set up a database from scratch.
+With Rails, we will be using PostgreSQL, a popular open-source database. Some PaaS providers like Heroku will automatically create a PostgreSQL database for our application when we first deploy. Others, like Fly.io, will have a few more manual steps involved, but still beat having to set up a database from scratch.
```

Comment:
Plural "beats" reads better to me here.

Reply:
Here's a third idea:
"… but it still beats having to set up …"

Here, the pronoun's antecedent isn't intended to be "others" (i.e., other PaaS providers) but instead the whole situation (i.e., having to do a few manual steps still beats setting up a database from scratch). I think this may be more inline with your original intention.

Reply:
Sounds perfect, lets do it